lib/dradis/plugins/veracode/importer.rb

```diff
@@ -25,11 +25,11 @@ def import(params={})
       end
 
       # create app_name, and parse attributes
-      parse_report_details(xml.root)
+      @node = parse_report_details(xml.root)
```

Comment:
can this be an argument we pass to #parse_flaw instead? Doesn't seem like a useful attribute (or related to the Importer) class.